tools/container.bzl

```
@@ -30,19 +29,13 @@ def multiarch_go_image(name, binary):
         tags = ["manual"],
     )
 
-    for arch in ["amd64", "arm64"]:
-        arch_image_target = "{}_{}_image".format(name, arch)
-        target_platform = "@rules_go//go/toolchain:linux_" + arch
-        images.append(arch_image_target)
-        platform_transition_filegroup(
-            name = arch_image_target,
-            srcs = [image_target],
-            target_platform = target_platform,
-        )
-
     oci_image_index(
         name = name,
-        images = images,
+        images = [image_target],
+        platforms = [
+            "@rules_go//go/toolchain:linux_" + arch
+            for arch in ["amd64", "arm64"]
+        ]
```

I would prefer these expanded as I think that is a bit more clear.
```suggestion
            "@rules_go//go/toolchain:linux_amd64",
            "@rules_go//go/toolchain:linux_arm64",
        },
```

---

done